### PR TITLE
Make draft email when email is clicked

### LINKED ISSUE #198: Make Chingu Support email clickable
The Chingu Support email displayed on the page should be clickable.

* A new email draft should be created / opened when clicking on the email
* The clickable email should have the same styling as other links on the page

### PR DESCRIPTION
**Pull request by leonyangela**
---
name: Pull Request 
about: When the user clicked support email, it will opens up mail app and create a draft email.
title: 'Make Chingu Support email clickable'
labels: 'Enhancement, good-first-issue, Front end'
assignees: 'leony'

---

When the user clicked support email, it will opens up mail app and create a draft email.

Resolves #198 

chingu-email-draft-feature

### CODE REVIEW

File: pages/data-privacy.tsx [hunk 1]
```diff
@@ -39,7 +45,7 @@ export default function DataPrivacy() {
 
         <br />
 
-        <p>If you you have questions about this privacy policy contact us at support@chingu.io</p>
+        <p>If you you have questions about this privacy policy contact us at <EmailTo as="a" href="mailto:support@chingu.io">support@chingu.io</EmailTo></p>
```
Comment (Soreg) [inline pages/data-privacy.tsx]: It's probably easier to just use an a-tag directly (`<a href="..."`) - so no new styled component needed.
We have default styles for all our a-tags, so we don't need to add custom hover-styles 🙂

**Review by Soreg — CHANGES_REQUESTED** [review]

File: pages/data-privacy.tsx (hunk 1 above)
Comment (leonyangela) [reply pages/data-privacy.tsx]: sure, will fix that

File: pages/data-privacy.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ const Section = styled.div`
   }
 `;
 
+
```
Comment (Soreg) [inline pages/data-privacy.tsx]: Last bit of change: I see an extra line was added here (after the styled component was removed). Please remove this one - then it's fine 👍

**Review by Soreg — CHANGES_REQUESTED** [review]

**Review by Soreg — APPROVED** [review]
Approved - good job 👍

**Review by cMikolai — APPROVED** [review]
Looks good!

**Pull request merged**